routes/user.js

```diff
@@ -0,0 +1,165 @@
+const router = require("express").Router();
+const bcrypt = require("bcryptjs");
+const Error = require("../models/Error");
+const Joi = require("@hapi/joi");
+const jwtService = require("../services/JwtService");
+
+const userService = require("../services/UserService");
+
+router
+    .route("/")
```

Comment:
We need to give it a proper route maybe? `api/users/` seems like we are `GET`ing a list of `users` rather than adding a new user. We need to have an endpoint like `api/users/add` or something like that.